src/backend/distributed/master/master_stage_protocol.c

```diff
@@ -636,7 +638,7 @@ UpdateShardStatistics(int64 shardId)
  * we assume have changed after new table data have been appended to the shard.
  */
 static bool
-WorkerShardStats(char *nodeName, uint32 nodePort, Oid relationId, char *shardName,
+WorkerShardStats(ShardPlacement *placement, Oid relationId, char *shardName,
```

Comment:
@lithp -- minor note. No need to take an action unless you strongly agree.

Now that we changed the function's signature, is ``PlacementShardStats()`` a better name for this function?